tests/Doctrine/Tests/Models/GH10334/GH10334ProductType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\GH10334;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\OneToMany;
+
+#[Entity]
+class GH10334ProductType
+{
+    #[Id]
+    #[Column(type: 'string', enumType: 'Doctrine\Tests\Models\GH10334\GH10334ProductTypeId', length: 255)]
```

Comment:
Should this be reversed after the MariaDB platform was fixed?

Reply:
Not sure what you mean :thinking: … is MariaDB broken in that it's not supposed to require a length? Also, why the past tense? Has it been fixed recently?

Reply:
I understand that the length was added because of a buggy-behaviour that the MariaDB DBAL platform introduced. Since this should actually work without the length too, I'd expect that the added length could be removed again once it was fixed in dbal.